packages/nextjs-mf/utils/build-utils.ts

```diff
@@ -3,9 +3,13 @@ import { parseRemoteSyntax } from '../src/internal';
 import { WebpackRemoteContainer } from '@module-federation/utilities';
 import path from 'path';
 
-const swc = require('@swc/core');
-
 const transformInput = (code: string) => {
+  let swc
+  try {
+    swc = require('@swc/core');
```

Comment:
causes file trace problems when using a custom server that imports next.config to create NextNodeServer on AWS